tests/integration/features/test_marks.py

```diff
@@ -0,0 +1,33 @@
+# vim: ft=python fileencoding=utf-8 sts=4 sw=4 et:
+
+# Copyright 2014-2016 Florian Bruhin (The Compiler) <mail@qutebrowser.org>
+#
+# This file is part of qutebrowser.
+#
+# qutebrowser is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# qutebrowser is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with qutebrowser.  If not, see <http://www.gnu.org/licenses/>.
+
+import pytest_bdd as bdd
+bdd.scenarios('marks.feature')
+
+
+def _get_scroll_y(quteproc):
+    data = quteproc.get_session()
+    pos = data['windows'][0]['tabs'][0]['history'][-1]['scroll-pos']
+    return pos['y']
+
+
+@bdd.then(bdd.parsers.re(r"the page should be scrolled to "
+                         r"(?P<y>\d+)"))
```

Comment:
You could simplify this to `@bdd.then(bdd.parsers.parse("the page should be scrolled to {y}"))` (with line breaks as appropriate), as the `int()` call will reject invalid values anyways.